src/main/kotlin/com/openlattice/data/storage/aws/AwsBlobDataService.kt

```diff
@@ -43,8 +53,10 @@ class AwsBlobDataService(
         metadata.contentLength = dataInputStream.available().toLong()
         metadata.contentType = contentType
         val putRequest = PutObjectRequest(datastoreConfiguration.bucketName, s3Key, dataInputStream, metadata)
-        s3.putObject(putRequest)
-
+        val transferManager = TransferManagerBuilder.standard().withS3Client(s3).build()
```

Comment:
I think you want a `.withMultiPartUploadThreshold(16_000_000L)` here

Reply:
I believe TransferManager defaults to do multipart upload at 16MB so it doesn't need to be explicitly set

Reply:
Ah yeah they don't put that in the comments anywhere in TransferManager or TMBuilder 🤦‍♂